### PR TITLE
copy .git/** into build container so git.properties can be generated [FINERACT-983]

### PR DESCRIPTION
**Pull request by vorburger**
read details in FINERACT-983

### CODE REVIEW

**Comment by vorburger** [comment]
@xurror @ptuomola @awasum check this out.. do you have any better ideas? Merge?

File: .dockerignore [hunk 1]
```diff
@@ -18,12 +18,9 @@
 build/*
 .gradle/*
 
+*.jar
```
Comment (xurror) [inline .dockerignore:21]: Why are we ignoring this?

**Review by xurror — APPROVED** [review]
This looks good to me

File: .dockerignore (hunk 1 above)
Comment (vorburger) [reply .dockerignore:21]: any *.jar that have been built locally should not get copied into the build container

Comment (xurror) [reply .dockerignore:21]: Ok, makes sense.

**Comment by vorburger** [comment]
@awasum Is this OK for you too?

**Comment by vorburger** [comment]
@ptuomola no objections to this? See linked JIRA. If @awasum or @xurror merge this ASAP, then I'll send an email to the dev list to announce the auto-updating of fineract.dev, which finally fully works end2end, since yesterday! I'd love this to be merged before I announce, because this change helps to "see" what's actually running on Fineract.dev.

**Comment by ptuomola** [comment]
Looks good - except the previous comment that copying .git slows things down, and that is why it was excluded. Is that no longer the case - ie do we not end up with the same issue now that we are again including .git? It's basically copying the entire repository history, right?

**Comment by awasum** [comment]
Can we get the CI system to copy just the latest revision of the history? Will that help?
Like:
`git clone -–depth [depth] [remote-url]`

**Comment by awasum** [comment]
Maybe we do this in a follow up PR? Or even rethink the issue of adding latest git revision in /actuator/info... IS there a better way?

**Comment by awasum** [comment]
I will merge this and someone can cycle back to this in the near future.

**Comment by vorburger** [comment]
https://demo.fineract.dev/fineract-provider/actuator/info - yay!! (This was fully automated.)

@ptuomola @awasum see my comments in FINERACT-983 and FINERACT-911 re. your points.

**Pull request merged**